docs/reference/commandline/ps.md

```diff
@@ -24,6 +24,7 @@ weight=1
       -q, --quiet=false     Only display numeric IDs
       -s, --size=false      Display total file sizes
       --since=""            Show created since Id or Name, include non-running
+      -F, --format=[]       Pretty-print containers using a Go template
```

Comment:
I disagree, it definitely matters as it's super confusing between `inspect` and `ps` options, these should match as much as possible and is just plain old bad UX.

If this is going into 1.8 as the milestone says then I think we should drop. If not we can certainly merge and discuss on another PR as this really is a UI issue.
If this is going into 1.8 as the milestone says then I think we should drop. If not we can certainly merge and discuss on another PR as this really is a UI issue.